test/interpreter/fixtures/integer-operations/module.wast

```diff
@@ -0,0 +1,172 @@
+  )
+  (export "div_u" (func $div_u))
+  (func $and (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.and)
+  )
+  (export "and" (func $and))
+  (func $rem_s (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.rem_s)
+  )
+  (export "rem_s" (func $rem_s))
+  (func $rem_u (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.rem_u)
+  )
+  (export "rem_u" (func $rem_u))
+  (func $shl (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.shl)
+  )
+  (export "shl" (func $shl))
+  (func $shr_s (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.shr_s)
+  )
+  (export "shr_s" (func $shr_s))
+  (func $shr_u (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.shr_u)
+  )
+  (export "shr_u" (func $shr_u))
+  (func $rotl (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.rotl)
+  )
+  (export "rotl" (func $rotl))
+  (func $rotr (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.rotr)
+  )
+  (export "rotr" (func $rotr))
+  (func $clz (param i32) (result i32)
+   (get_local 0)
+   (i32.clz)
+  )
+  (export "clz" (func $clz))
+  (func $ctz (param i32) (result i32)
+   (get_local 0)
+   (i32.ctz)
+  )
+  (export "ctz" (func $ctz))
+  (func $popcnt (param i32) (result i32)
+   (get_local 0)
+   (i32.popcnt)
+  )
+  (export "popcnt" (func $popcnt))
+  (func $eqz (param i32) (result i32)
+   (get_local 0)
+   (i32.eqz)
+  )
+  (export "eqz" (func $eqz))
+  (func $eq (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.eq)
+  )
+  (export "eq" (func $eq))
+  (func $ne (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.ne)
+  )
+  (export "ne" (func $ne))
+  (func $lt_s (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.lt_s)
+  )
+  (export "lt_s" (func $lt_s))
+  (func $lt_u (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.lt_u)
+  )
+  (export "lt_u" (func $lt_u))
+  (func $le_s (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.le_s)
+  )
+  (export "le_s" (func $le_s))
+  (func $le_u (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.le_u)
+  )
+  (export "le_u" (func $le_u))
+  (func $gt_s (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.gt_s)
+  )
+  (export "gt_s" (func $gt_s))
+  (func $gt_u (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.gt_u)
+  )
+  (export "gt_u" (func $gt_u))
+  (func $ge_s (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.ge_s)
+  )
+  (export "ge_s" (func $ge_s))
+  (func $ge_u (param i32) (param i32) (result i32)
+   (get_local 0)
+   (get_local 1)
+   (i32.ge_u)
+  )
+  (export "ge_u" (func $ge_u))
+)
```

Comment:
I really like that, the spec is way more testable this way, but way less maintainable.